### ISSUE TITLE
copyOf malformed javadoc

### ISSUE THREAD

**Issue by elucash** [immutables/immutables#121]
FROM THE MAILING LIST:

It seems all the other generated methods include `@param` for all parameters, but not `copyOf`:
```
:javadoc

.../ImmutableSD.java:187: warning: no @param for instance
  public static ImmutableSD copyOf(SD instance) {
                            ^
```
Right now I have to special case `Immutable*` in my build to get around it.  It would be good to have it just work without special casing it.

**Comment by haroldcarr** [comment]
Any chance of getting this in a point release soon?

**Comment by elucash** [comment]
Later today or tomorrow I'll release it, there are some other changes worthwhile to be released (#119, #120)

**Comment by haroldcarr** [comment]
Great - thanks!

**Comment by elucash** [comment]
Released 2.0.14!

**Comment by haroldcarr** [comment]
+1 !
